wyvern/components/ranking_pipeline.py

```diff
@@ -67,13 +69,16 @@ class RankingResponse(BaseModel):
         events: the list of logged events
     """
 
-    ranked_candidates: List[ResponseCandidate]
+    ranked_products: List[ResponseCandidate]
```

Comment:
Just curious: Why is this `ranked_products`? Candidate is a generic term, but product is specific to a type of entity

Reply:
Inventa has been using the `ranked_products` as the schema for all of the use cases. We don't want them to do a API migration on their server side any more.  i can also build pydantic "alias" for this field 🤔

Reply:
oh. Is it possible to decouple this object from the actual response object? Maybe we can re-name it at the very end or something like that.

They'll definitely want `ranked_products` or `ranked_brands` depending on the pipeline.